sdk/documenttranslation/azure-ai-documenttranslation/tests/preparer.py

```diff
@@ -0,0 +1,46 @@
+
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import os
+import functools
+from devtools_testutils import PowerShellPreparer, AzureMgmtPreparer
+from azure.core.credentials import AzureKeyCredential
+
+
+DocumentTranslationPreparer = functools.partial(
+    PowerShellPreparer,
+    'documenttranslation',
+    documenttranslation_test_endpoint="https://redacted.cognitiveservices.azure.com/",
+    documenttranslation_test_api_key="fakeZmFrZV9hY29jdW50X2tleQ==",
+    documenttranslation_name="redacted",
```

Comment:
is this the name of the document?

Reply:
no, that's here: https://github.com/Azure/azure-sdk-for-python/blob/7770b87a13d71f73db33281d01257235f1abab66/sdk/documenttranslation/azure-ai-documenttranslation/tests/testcase.py#L91

Reply:
I see, so what is this name?

Reply:
It ends up being something like "ccda6168-42dd-4239-80f2-0aecf398e94f.txt" if you don't add a prefix